package.json

```
@@ -7,19 +7,19 @@
     "test": "tests"
   },
   "dependencies": {
-    "chai": "^4.2.0",
-    "mocha": "^5.2.0"
   },
   "devDependencies": {
+    "chai": "^4.2.0",
     "eslint": "^5.6.0",
     "eslint-config-standard": "^12.0.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0"
+    "eslint-plugin-standard": "^4.0.0",
+    "mocha": "^5.2.0"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha --recursive"
```

```suggestion
    "test": "mocha --recursive",
    "lint": "$(npm bin)/eslint ."
```

Run this with `npm run lint`